cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/PreviewRunnerTwillRunnable.java

```diff
@@ -219,7 +220,8 @@ protected void configure() {
     modules.add(new PreviewRunnerManagerModule().getDistributedModules());
     modules.add(new DataSetServiceModules().getStandaloneModules());
     modules.add(new DataSetsModules().getStandaloneModules());
-    modules.add(new AppFabricServiceRuntimeModule().getStandaloneModules());
+    modules.add(Modules.override(new AppFabricServiceRuntimeModule(cConf).getStandaloneModules())
```

Comment:
Does it make sense to move this override to `AppFabricServiceRuntimeModule` (e.g. `#getPreviewModules`)? This way it's more clear what is added as it's in same file

Reply:
Fixed